src/QRCodeDataSchema.ts

```
@@ -24,46 +24,44 @@ import {QRCode} from '../index';
+   *
+   * @var {number}
+   */
+  public readonly VERSION = 3;
 
-    /**
-     * The AccountQR QR Code version
-     *
-     * @var {number}
-     */
-    public readonly VERSION = 3;
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)